app/furniture/marketplace/carts/_footer.html.erb

```diff
@@ -1,9 +1,16 @@
+<% 
+  marketplace = cart.marketplace
+  room = marketplace.room
+  space = room.space
+%>
+
 <tfoot id="cart-footer-<%= cart.id%>" class="bg-gray-50">
   <tr>
     <td></td>
+    <th> <%= link_to("checkout", new_space_room_marketplace_checkout_path(space, room, marketplace), html_options = {}) %> </th>
```

Comment:
We may be able to use `[:new, space, room, marketplace, :checkout]` in place of `new_space_room_marketplace_checkout_path(space, room, marketplace)`